app/views/layouts/head/_scripts.html.erb

```diff
@@ -1,12 +1,17 @@
-<!-- Scripts -->
-<%= javascript_include_tag 'application' %>
-<% if @ext_js.present?
-  Array.wrap(@ext_js).each do |js|
-    concat "  #{javascript_include_tag js}\n".html_safe
-  end
-end %>
-<% if @inline_js.present?
-  Array.wrap(@inline_js).each do |js|
-    concat "  <script>\n#{js}\n  </script>\n".html_safe
-  end
-end %>
+  <!-- Scripts -->
+  <%= javascript_include_tag 'application' %>
+  <% if @ext_js.present? || @inline_js.present? %>
+  <!-- Conditional -->
```

Comment:
Oh, it is just mean to display the additional `<!-- Conditional -->` HTML comment to distinguish which scripts/styles were conditionally loaded for the given page if one looks at the page source. In case we want to check when debugging a page's JS or CSS, "Does this page have conditional assets?" it's a quick place to look for that. I hadn't intended it to be anything more than that. Did you have other ideas for it in overrides?